scss/components/_blog.scss

```diff
@@ -48,31 +63,74 @@
   @include flex;
 }
 
+.right-blog-arrow {
+  margin-left: 90%;
+  border-left: none;
+  border-right: 2px solid $data-color;
+  transform: rotate(315deg);
+  @extend %arrow;
+
+  &:hover {
+    border-right: 2px solid $second-color;
+    border-bottom: 2px solid $second-color;
+    border-left: none;
+  }
+}
+
+.left-blog-arrow {
+  @extend %arrow;
+  width: 28px;
+  height: 28px;
+  border-left: 2px solid $data-color;
+  border-bottom: 2px solid $data-color;
+  transform: rotate(45deg);
+  left: 3%;
```

Comment:
You have already had styles `left`, 'transform', `border` at `%arrow` template, haven't you?